.github/workflows/build.yml

```diff
@@ -82,7 +82,10 @@ jobs:
         with:
           python-version: "${{ matrix.python }}"
       - name: Install python dependencies
-        run: make setup-docs
+        run: |
+          pip install -r requirements.txt
```

Comment:
Is this line needed?

I ask because I believe readthedocs will not install from our top-level `requirements.txt`, so unless this is needed for some other purpose, it would be more accurate to omit it.

Also, since we are all in on Makefiles now, it might be nice to keep this behavior under `make setup-docs`.

Reply:
yea, good catch, this line of pip install is not necessary, so we can take that away.

I thought something funky disallowed me to write setup-docs in makefile, but it turned out that some tab-space issue in makefile. I will get it fixed.